tests/integration/commands/dev/redirects.test.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, expect, test } from 'vitest'
+
+import { FixtureTestContext, setupFixtureTests } from '../../utils/fixture.js'
+import got from '../../utils/got.cjs'
+
+describe('redirects', () => {
+  setupFixtureTests('dev-server-with-functions', { devServer: true }, () => {
+    test<FixtureTestContext>('should send original query params to functions', async ({ devServer }) => {
+      const response = await got(`http://localhost:${devServer.port}/with-params?param2=world&other=1`, {
+        throwHttpErrors: false,
+        retry: { limit: 0 },
+      })
+
+      expect(response.statusCode).toBe(200)
+
+      const result = JSON.parse(response.body)
+      expect(result.queryStringParameters).not.toHaveProperty('param1')
+      expect(result.queryStringParameters).toHaveProperty('param2', 'world')
+      expect(result.queryStringParameters).toHaveProperty('other', '1')
```

Comment:
@paulo is this the expected behavior? Just to double-check. So `param1` even though it is specified in the redirect is not forwarded.